pkg/models/dashboard_version.go

```diff
@@ -70,6 +71,7 @@ type GetDashboardVersionQuery struct {
 
 type GetDashboardVersionsQuery struct {
 	DashboardId int64
+	UID         string
```

Comment:
Same as above; I would suggest better rename it to DashboardUID to be clearer.
```suggestion
	DashboardUID         string
```